device/samples/javascript/dmpatterns_reboot_device.js

```diff
@@ -2,9 +2,14 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 'use strict';
 
-var Client = require('azure-iot-device').Client;
+// Choose a protocol by uncommenting one of these transports.
```

Comment:
Do we want to leave all of them commented so we force the customer to make a choice? I'm not sure about this myself but something to consider.

Reply:
i think we should default to having mqtt enabled. If the user wants to try another choice they can comment and uncomment. I would like to push people to use mqtt